supabase/functions/_shared/exception/ValidationError.ts

```
@@ -0,0 +1,7 @@
+class ValidationError extends Error {
+  constructor(message: string) {
+    super(message)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)